charts/pega/values-large.yaml

```diff
@@ -87,6 +87,9 @@ global:
       url: "YOUR_DOCKER_REGISTRY"
       username: "YOUR_DOCKER_REGISTRY_USERNAME"
       password: "YOUR_DOCKER_REGISTRY_PASSWORD"
+    # To avoid exposing docker registry details, create secrets for docker registry credentials.
+    # Specify secrets as an array of comma-separated strings using the image_pull_secret_names parameter. e.g. ["secret1", "secret2"]
+    image_pull_secret_names: []
```

Comment:
Can you use CamelCase names?

Reply:
Updated naming convention for image_pull_secret_names.